src/lib/symlink-manager.ts

```diff
@@ -195,6 +210,9 @@ export class SymlinkManager {
 			vfsTarget,
 			createNodeFsMountHandler( hostTarget )
 		);
+		// For some reason, we must access the target directory after mounting it, otherwise the
+		// mount will not work.
+		this.php.listFiles( vfsTarget );
```

Comment:
@bgrgicak @adamziel @ashfame, I'm not sure why this is needed, but the Yoast SEO test case (see PR description) didn't work without it in my testing. Any idea why that's the case..?

Reply:
It continues to work for me when I comment out this line. Maybe it depends on the underlying filesystem and we use different ones?

Reply:
To quickly double-check: did you restart the entire app after commenting out this line?

Reply:
I was able to mount and start a site with `wp-data-layer-app-pages` plugin without listing the files.

Reply:
Yeah, the `wp-data-layer-app-pages` test case works without this line for me, too. It's only the Yoast test case that was troublesome